File: e2e-tests/spring-examples/src/main/java/com/foo/rest/examples/spring/db/directint/DbDirectIntApplication.java
package com.foo.rest.examples.spring.db.directint;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Bean;
import springfox.documentation.builders.ApiInfoBuilder;
import springfox.documentation.service.ApiInfo;
import springfox.documentation.spi.DocumentationType;
import springfox.documentation.spring.web.plugins.Docket;
import springfox.documentation.swagger2.annotations.EnableSwagger2;

import static springfox.documentation.builders.PathSelectors.regex;

@EnableSwagger2
@SpringBootApplication
public class DbDirectIntApplication {

    public static void main(String[] args) {
        SpringApplication.run(DbDirectIntApplication.class, args);
    }

    @Bean
    public Docket piApi() {
        return new Docket(DocumentationType.SWAGGER_2)
                .apiInfo(apiInfo())
                .select()
                .paths(regex("/api/db/directint.*"))
                .build();
    }

    private ApiInfo apiInfo() {
        return new ApiInfoBuilder()
                .title("API")
                .description("Some description")
                .version("1.0")
                .build();
    }
}

File: e2e-tests/spring-graphql/src/main/kotlin/com/foo/graphql/db/directint/DbDirectIntApplication.kt
package com.foo.graphql.db.directint

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
open class DbDirectIntApplication{
    companion object{
        const val SCHEMA_NAME = "dbdirectInt.graphqls"
    }
}

fun main(args: Array<String>) {
    SpringApplication.run(DbDirectIntApplication::class.java,
        "--graphql.tools.schema-location-pattern=**/${DbDirectIntApplication.SCHEMA_NAME}")
}